openslides/mediafiles/serializers.py

```diff
@@ -1,14 +1,38 @@
-from openslides.utils.rest_api import ModelSerializer, SerializerMethodField
+import mimetypes
 
+from django.db import models as dbmodels
+from rest_framework.fields import FileField
```

Comment:
See `openslides/utils/rest_api.py`. All openslides apps import rest_framework stuff from this file rather than from the rest_framework module directly. So we can override these classes later if necessary. We want to do this only for rest_framework stuff at the moment because we do not know where the travel goes to ... :-)